coverage/cmdline.py

```diff
@@ -85,6 +85,10 @@ class Opts(object):
             "which isn't done by default."
         ),
     )
+    sort = optparse.make_option(
+        '--sort', action='store', metavar='COLUMN',
+        help="Sort the report by the named column"
```

Comment:
Let's add the choices here:
```suggestion
        help="Sort the report by the named column: name, stmts, miss, branch, brpart, or cover. Default is name."
```